tools/topology/m4/utils.m4

```diff
@@ -51,7 +51,7 @@ define(`COMP_FORMAT_NAME',
 
 dnl P_GRAPH(name, CONNECTIONS)
```

Comment:
I think we can also make the function comment changed too.
```suggestion
dnl P_GRAPH(name, pipeline id, CONNECTIONS)
```

Reply:
This seems to be wrong before this patch already, but anyways an important correction.

Reply:
@xiulipan @kv2019i done!